python/src/equistore/operations/reduce_over_samples.py

```diff
@@ -222,33 +237,33 @@ def _reduce_over_samples_block(
 
 
 def _reduce_over_samples(
-    tensor: TensorMap, samples_names: Union[List[str], str], reduction: str
+    tensor: TensorMap, sample_names: Union[List[str], str], reduction: str
 ) -> TensorMap:
     """
     Create a new :py:class:`TensorMap` with the same keys as as the input
     ``tensor``, and each :py:class:`TensorBlock` is obtained summing the
-    corresponding input :py:class:`TensorBlock` over the ``samples_names``
+    corresponding input :py:class:`TensorBlock` over the ``sample_names``
     indices.
 
-    "sum", "mean", "std" or "variance" reductions can be performed.
+    "sum", "mean", "std" or "var" reductions can be performed.
```

Comment:
I would leave this as `variance`, saving 3 characters here makes the documentation harder to read

Reply:
So, this was deliberate. These mirror array operations in `numpy` and `torch` , and it is kind of inconsistent to use the same names for sum, mean, std and then use `variance` where numpy and torch use `var`.

Reply:
Ok, fair enough